upper-constraints.txt

```diff
@@ -431,7 +431,7 @@ xmltodict===0.13.0
 pyasn1===0.4.8
 directord===0.12.0
 oslo.rootwrap===7.0.1
-Django===3.2.16
+Django===3.2.19
```

Comment:
Horizon requirement in 2023.1 is
```
Django>=3.2,<3.3 # BSD
```
So in theory it should work. I haven't been through all of the UI plugins though, they could potentially have different requirements. Hopefully pip install would fail if so though.